README.md

```diff
@@ -8,6 +8,15 @@ but also make certain assumptions that you may not want to impose on your module
 [LabKey documentation](https://www.labkey.org/Documentation/wiki-page.view?name=gradleModules) for more information.
 
 ## Release Notes
+### version x.y.z
+*Released*: xxx
+(Earliest compatible LabKey version: 20.3)
+
+* Rename "version" property to "schemaVersion" and support null value
+* Rename "labkeyVersion" property to "releaseVersion"
+* Remove "consolidateScripts" property
```

Comment:
Can you qualify these comments with something like "Change module properties/module.xml writer"?

Reply:
Updated